Cairo Desktop/Cairo Desktop/MenuBar.xaml.cs

```diff
@@ -863,7 +863,7 @@ private void SysSleep(object sender, RoutedEventArgs e)
         private void InitCairoSettingsWindow(object sender, RoutedEventArgs e)
         {
             CairoSettingsWindow window = new CairoSettingsWindow();
-            window.Show();
+            window.ShowDialog();
```

Comment:
ShowDialog() is a blocking operation and should be using Show() as it was previously.